GETTING_STARTED.md

```diff
@@ -0,0 +1,62 @@
+
+## Overview
+
+### The Project
+The Slippi project is comprised of a number of different applications, each with their own purpose. Below is an overview of each of these applications function, and their relevant technologies.
+
+<b> [Ishiiruka](https://github.com/project-slippi/Ishiiruka) </b> - A modified version of the Dolphin emulator. This project is responsible for handling things like: communication with the matchmaking server, writing Slippi replay files, passing external data to the emulated game, and playing replays. 
+<br> <i> Languages: </i> C++ 
+
+<b> [Slippi SSBM ASM](https://github.com/project-slippi/slippi-ssbm-asm) </b> - A series of ASM mods that are applied to Melee in order make Slippi work.
+<br> <i> Languages: </i> PPC Assembly
+
+<b> [Slippi Desktop App](https://github.com/project-slippi/slippi-desktop-app) </b> - A desktop application that allows users to view statstics about previous matches, and launch replays.
+<br> <i> Languages: </i> JavaScript
+<br> <i> Frameworks: </i> Electron, React
+
+<b> [Slippi-JS](https://github.com/project-slippi/slippi-js) </b> - A JavaScript library that is used to parse .slp files, and allows for the calculation of game statistics. 
+<br> <i> Languages: </i> JavaScript
+
+### The Workflow
+
+The user launches <b>Ishiiruka</b> and selects an .iso of Melee to emulate. Upon lauching the emulation of Melee, Ishiiruka injects the modifications made by <b> Slippi SSBM ASM </b>. As the user interacts with the game, information is exchanged between Ishiiruka and the Slippi SSBM ASM code. As a user begins an online match, Ishiiruka starts a log of in-game state reported by Slippi SSBM ASM. This log is ultimately formatted and written to a .slp file, according to the [Slippi replay file spec](https://github.com/project-slippi/slippi-wiki). These files are then viewable from the <b> Slippi Desktop App </b>. From the desktop app, a user may choose to launch the replay, in which case Ishiiruka is launched in a replay mode. A user may also choose to view details or statistics about a previously played match, in order to do so, the desktop app leverages the <b> Slippi-JS </b> library.  
+
+
+## Commonly Asked Questions
+> "Where is the rollback code located?"  
+
+Rollback is accomplished by work done between the [Slippi SSBM Asm](https://github.com/project-slippi/slippi-ssbm-asm/search?p=1&q=rollback&unscoped_q=rollback) code and the [Ishiiruka](https://github.com/project-slippi/Ishiiruka/search?q=rollback&unscoped_q=rollback) code.
+
+> "How is data moved between the game (assembly) and the emulator?"
+
+Via EXI communication. An example of such is demonstrated in [this video](https://github.com/project-slippi/Ishiiruka/search?q=rollback&unscoped_q=rollback) by Fizzi.
+
+> "Where is the matchmaking code?"
+
+The source code for the matchmaking server is in a private repository, and is not currently planned to be open sourced.
+
+## Guides & Resources
+
+### Melee Modding and Assembly
```

Comment:
maybe add a link to the IBM PPC reference sheet

Reply:
Just updated. I think I included the reference sheet you're referring to. In any case, I found it pretty informative.

Reply:
I was thinking of this one:

https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/assembler/idalangref_ins_set.html